lmdeploy/vl/model/internvl.py

```diff
@@ -32,16 +94,58 @@ def build_model(self):
         load_model_from_weight_files(model, self.model_path)
         self.model = model
         self.model.to(self.device).eval()
-        self.image_processor = CLIPImageProcessor.from_pretrained(
-            self.model_path)
+        self.config = config
 
-    @torch.no_grad()
-    def forward(self, images: List[Image]) -> List[torch.Tensor]:
-        """forward."""
+        if getattr(self.config, 'dynamic_image_size', False):
+            logger.info('using InternVL-Chat-V1-5 vision preprocess')
+            MEAN = (123.675, 116.28, 103.53)
+            STD = (58.395, 57.12, 57.375)
```

Comment:
The two constants, can it be inferred from the Internvl code?

Reply:
Let's try not to infer any thing from the upstream's repo code. We'd better keep them as independent as possible

Reply:
the mean and std are not in the repo the in the example code.